<launch>
    <!-- global params -->
    <param name="/use_sim_time" value="false"/>

    <node name="depthmap" pkg="hole_tracker" type="node_depthmap.py" output="screen">
        <!-- params -->
        <param name="run_hz" value="30"/>

        <!-- input topics -->
        <remap from="input_pcl" to="/quail/normal_estimation/pico_flexx/points"/>

        <!-- output topics -->
        <remap from="output_depthmap" to="/tracker/depth_map/compressed"/>

    </node>
</launch>